Database/mssql/scripts/init_tables.sql

```diff
@@ -0,0 +1,19 @@
+USE Info_Skjerm;
+
+CREATE TABLE Users
+(
+    email VARCHAR(100) PRIMARY KEY,
+    fullname VARCHAR(100),
+    password_hash TEXT,
+);
+
+CREATE TABLE Events
+(
+    ID INT IDENTITY(1,1) PRIMARY KEY,
+    title VARCHAR(100),
+    body TEXT,
+    startdate DATE,
+    starttime TIME,
+    enddate DATE,
+    endtime TIME,
```

Comment:
Æ d nødvendig å ha `startdate` og `starttime`? Æ d it nok mæ bare en `startdate`som inneheld startdate og starttime i d hær formatet `2024-04-29T11:00:00Z`?